print.css

```diff
@@ -0,0 +1,87 @@
+@charset "UTF-8";
+/*
+Theme Name: Twenty Nineteen
+
+Adding print support. The print styles are based on the the great work of
+Andreas Hecht in https://www.jotform.com/blog/css-perfect-print-stylesheet-98272/.
+*/
+/*--------------------------------------------------------------
+>>> TABLE OF CONTENTS:
+----------------------------------------------------------------
+# Margins
+# Typography÷
+# Page breaks
+# Links
+# Visibility
+--------------------------------------------------------------*/
+@media print {
+  /* Margins */
+  @page {
+    margin: 2cm;
+  }
+  .hentry, .hentry .entry-header, .site-footer .site-info {
```

Comment:
Do not style `hentry`!  There is already an issue to remove styles for `hentry`.